### DISCUSSION TITLE
steps after netlify deployment

### DISCUSSION THREAD

**Discussion by shaddu007** [PublicAffairs/openai-gemini#7] [Q&A] [answered]
can any one tell me what to to do after netlify deployment where it is showing 404 error, where i need to gemini api key ?
can any open tell me step wise?

**Comment by johnd0e** [comment]
See README.

How to use

If you open your newly-deployed site in a browser, you will only see a 404 Not Found message. This is expected, as the API is not designed for direct browser access. To utilize it, you should enter your API address and your Gemini API key into the corresponding fields in your software settings.

...

**Reply by shaddu007** [reply]
Yes im new here i just click on the netify deploye button on the readme file after that it ask me to give access to my github and automatically deployed the site.. Can you please elaborate what to do after that i will be really helpfull for me. because i dont understand where i need to change in netify deployement etc

**Reply by johnd0e** [reply, marked as answer]
You do not need to change anything in your Netlify deployment.

Please read the whole README.md again, and then write me what exactly is not clear there, and I will try to explain.
